### ISSUE TITLE
Selecting the 'Settings' button for axes > 8 crashes application

### ISSUE THREAD

**Issue by rjwills28** [Observatory-Sciences/dls-pmac-control#4]
An issue recently seen is that although it's OK to click the settings button for axis 1-8, if you then go to axis 9 or beyond where these axes are normally implemented in a macro accessory card such as ACC24 or Geomacro drive then the application bombs out.

**Comment by rjwills28** [comment]
I have fixed this issue where the control application crashes when trying to select 'Settings' for an axis > 8. I believe the application was crashing because the pmac was returning 'ERR008' instead of the requested I variable values. I have added a catch to ensure that the app does not crash in this case.

The 'ERR008' error code indicates that there was a 'MACRO auxiliary communications error' where the solution is 'should check MACRO ring communications error hardware and software setup' (taken from the manual).

In this case I have implemented the following behaviour:

- App will not crash (always a good thing),
- The fields for the values in the Setting window will be empty (as I think is the case with the original version of dls-pmac-control).
- A small dialog box will pop-up with the error to inform the user that there is a set-up issue.

**Comment by rjwills28** [comment]
There is a separate issue with this one in that sometime axis 9 will be mapped to MS4 instead of the assumed MS0. In order to accommodate this I have added a command line option to set the macro axis starting index. In this example, running the application with '-m 4' will offset the queried macro offset by 4 (hence using MS4 instead of MS0). If no argument is provided then '0' will be assumed.
Usage:
```
    dls-pmac-control.py -m <number>
    dls-pmac-control.py --macroAxisStartIndex <number>
```
Commits to library and application

**Comment by rjwills28** [comment]
Fix is available in the released v3.0.0.
